getting_started/create_vm/make_vm.sh

```diff
@@ -10,6 +10,11 @@ RG=ccf-dev
 REGION=eastus
 CFG=vm.json
 
+# By default, use current ssh public key
```

Comment:
it's always good for a script to print out any default's it is using

Reply:
It should print it already (`set -x` at the top of the file).